src/plugins/d3/__stories__/line/Shapes.stories.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import {StoryObj} from '@storybook/react';
+import {Button} from '@gravity-ui/uikit';
+import {settings} from '../../../../libs';
+import {D3Plugin} from '../..';
+import {ChartKitWidgetData, LineSeriesData} from '../../../../types';
+import {ChartKit} from '../../../../components/ChartKit';
+import nintendoGames from '../../examples/nintendoGames';
+import {HighchartsPlugin} from '../../../highcharts';
+
+enum LineShapeType {
+    Solid = 'Solid',
+    ShortDash = 'ShortDash',
+    ShortDot = 'ShortDot',
+    ShortDashDot = 'ShortDashDot',
+    ShortDashDotDot = 'ShortDashDotDot',
+    Dot = 'Dot',
+    Dash = 'Dash',
+    LongDash = 'LongDash',
+    DashDot = 'DashDot',
+    LongDashDot = 'LongDashDot',
+    LongDashDotDot = 'LongDashDotDot',
+}
+const SHAPES_ORDER = {
+    [LineShapeType.Solid]: 1,
+    [LineShapeType.Dash]: 2,
+    [LineShapeType.Dot]: 3,
+    [LineShapeType.ShortDashDot]: 4,
+    [LineShapeType.LongDash]: 5,
+    [LineShapeType.LongDashDot]: 6,
+    [LineShapeType.ShortDot]: 7,
+    [LineShapeType.LongDashDotDot]: 8,
+    [LineShapeType.ShortDash]: 9,
+    [LineShapeType.DashDot]: 10,
+    [LineShapeType.ShortDashDotDot]: 11,
+};
```

Comment:
If this is a utility for automatic distribution of forms for series, then it is better to move it somewhere inside. So that it can be used outside, for example. Not just for stories

Reply:
No, it should not be in ChartKit. ChartKit provides only an option to set dashStyle. This order of shapes is datalens-specific. And it is shown here just as good example for story.